SoftLayer/utils.py

```diff
@@ -406,3 +406,24 @@ def trim_to(string, length=80, tail="..."):
         return string[:length] + tail
     else:
         return string
+
+
+def format_comment(comment, max_line_length):
```

Comment:
Set a default value for `max_line_length`, maybe 60 characters here would be good as a default.